.travis.yml

```diff
@@ -112,27 +112,27 @@ before_install:
   - if [[ $BUILD == autotools ]]; then AUTOTOOLS=true; fi
   - if [[ $BUILD == cmake ]]; then CMAKE=true; fi
 
-  - if [[ $TRAVIS_BRANCH == master ]] ; then MASTER=true; fi
-
   # Set xvfb for headless qucs GUI test
   - "export DISPLAY=:99.0"
   -  if [[ $LINUX ]]; then sh -e /etc/init.d/xvfb start; fi
 
 install:
   # Setup Linux
   # Set up for Coveralls for GCC (LINUX) only
-  - if [[ $GCC ]]; then pip install --user cpp-coveralls --quiet; fi
+  - if [[ $GCC ]]; then pip3 install cpp-coveralls --quiet; fi
 
   # Qucs-Test uses Numpy to compare results
-  - if [[ $LINUX ]]; then pip install --user --only-binary=numpy,matplotlib numpy matplotlib --quiet; fi
+  - if [[ $LINUX ]]; then pip3 install --only-binary=numpy,matplotlib numpy matplotlib --quiet; fi
 
   # Setup OSX
   # Installed already: autoconf automake libtool pkg-config
   - if [[ $OSX ]]; then
       export PATH="/usr/local/opt/flex/bin:$PATH";
       export PATH="/usr/local/opt/bison/bin:$PATH";
-      pip2 install matplotlib;
+      pip3 install matplotlib;
       export PATH="/usr/local/opt/ccache/libexec:$PATH";
+      brew install ccache
```

Comment:
Travis is not running, you need to add semicolon (;) to the end of this line

Reply:
Thx, I don't have any messages from travis in such a case